exampleSite/utilities/examples/content/events/yyyy-city/conduct.md

```diff
@@ -1,6 +1,7 @@
 +++
 Title = "Conduct"
 Type = "event"
+Description = ""
```

Comment:
Okay I think I understand. Yeah, let's put sane default content in place so we don't have to prompt people. Nothing will stop them from editing it. I'll give that a shot.